File: CursoPOOUber/Java/Tarjeta.java
package Java;

public class Tarjeta extends Payment {
    Integer id;
    String numTarjeta;
    Integer cvv;
    float monto;
    String fecha;
    String email;
    
    public Tarjeta(String numTarjeta, Integer cvv,  float monto, String fecha, String email){
        super(monto, fecha, email);
        this.numTarjeta = numTarjeta;
        this.cvv = cvv;
        this.monto = monto;
        this.fecha = fecha;
        this.email = email;
    }
}

File: CursoPOOUber/PHP/tarjeta.php
<?php 

class Tarjeta extends Car{

    public $id;
    public $numTarjeta;
    public $cvv;
    public $monto;
    public $fecha;
    public $email;

    public function __construct($numTarjeta, $cvv, $monto, $fecha, $email)
    {
        $this->numTarjeta = $numTarjeta;
        $this->cvv = $cvv;
        $this->monto = $monto;
        $this->fecha = $fecha;
        $this->email = $email;
    }
}

?>